appveyor.yml

```diff
@@ -17,8 +17,8 @@ install:
 - reg ADD "HKLM\Software\Wow6432Node\Microsoft\StrongName\Verification\Microsoft.PackageManagement.NuGetProvider,31bf3856ad364e35"  /f
 
 
-build:
-  verbosity: normal
+build_script:
+  - msbuild "c:\projects\NuGetTest\NuGetProvider\vs-csproj\NugetLightProvider.csproj" /verbosity:normal /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
Current csproj defaults to Release/AnyCPU. Leaving the command line similar to the original. (not explicitly specifying configuration property)